docs/DeveloperGuide.md

```diff
@@ -1225,22 +1226,143 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### 7.2 Adding a patient
 
-### 7.2 Deleting a patient
+1. Add a patient while all patients are being shown
+
+   1. Prerequisites: List all patients using the `list` command. Multiple patients in the list.
+
+   1. Test case: `add id/ 54321 n/ John Doe p/ John f/ Curry chicken c/ Stable h/ Singing karaoke t/ amnesia`<br>
+      Expected: New patient is added to the list. Status message shows details of the new patient.
+
+      1. Other incorrect add commands to try: `add`, `add n/`<br>
```

Comment:
Would it be better if we include more incorrect command test cases such as missing required field(s)? 
Eg: `add id/ 54321 h/ Singing karaoke`